experimental/packages/propagator-aws-xray-lambda/test/AWSXRayLambdaPropagator.test.ts

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as assert from 'assert';
+
+import {
+  defaultTextMapGetter,
+  defaultTextMapSetter,
+  ROOT_CONTEXT,
+  SpanContext,
+  TraceFlags,
+  trace,
+  context,
+} from '@opentelemetry/api';
+import * as sinon from 'sinon';
+
+import { AWSXRAY_TRACE_ID_ENV_VAR, AWSXRayLambdaPropagator } from '../src';
+import {
+  AWSXRAY_TRACE_ID_HEADER,
+  AWSXRayPropagator,
+} from '@opentelemetry/propagator-aws-xray';
+
+describe('AWSXRayPropagator', () => {
+  const xrayLambdaPropagator = new AWSXRayLambdaPropagator();
+
+  let carrier: { [key: string]: unknown };
+
+  beforeEach(() => {
+    carrier = {};
+  });
+
+  describe('.inject()', () => {
+    it('should call use AWSXRayPropagator inject()', () => {
+      const spy = sinon.spy(AWSXRayPropagator.prototype, 'inject');
+      assert.equal(spy.callCount, 0);
+      xrayLambdaPropagator.inject(
+        context.active(),
+        carrier,
+        defaultTextMapSetter
+      );
+      assert.equal(spy.callCount, 1);
```

Comment:
Since that method is just a proxy, should we also assert that is was passed through 1:1?  (`sinon.assert.calledOnceWithExactly(..)`)

Reply:
Updated